app/models/sensor.rb

```diff
@@ -10,6 +10,7 @@ class Sensor < ActiveRecord::Base
   delegate :max, to: :sensor_type
   delegate :unit, to: :sensor_type
   delegate :fractionDigits, to: :sensor_type
+  delegate :data_collection_method, :manual?, :active?, to: :sensor_type
```

Comment:
I see, but then you can do `Sensor.includes(:sensor_type).order('"sensor_types"."manual"')`..

Reply:
or `joins` instead of `includes` maybe